templates/map.html.twig

```diff
@@ -0,0 +1,7 @@
+{% if map_token is defined and map_token != 'your_mapbox_access_token' %}
+    {% set apiConfig = {'type': 'FeatureCollection', 'features': [{'type': 'Feature', 'properties': {'marker-color': 'f7d325'}, 'geometry': {'type': 'MultiPoint', 'coordinates': conferenceCoordinates} }] } %}
+    {% set encode_apiConfig = apiConfig|json_encode|url_encode %}
+    {% set url = 'https://api.mapbox.com/styles/v1/mapbox/streets-v11/static/geojson(%s)/auto/1000x600?access_token=%s'|format(encode_apiConfig, map_token) %}
```

Comment:
This block a bit ugly. I would create a small and simple twig function for that.